src/components/_input.scss

```diff
@@ -45,3 +45,9 @@
     box-shadow: 1px solid $c-error;
   }
 }
+
+
+@mixin sui-focus() {
+  box-shadow: $bxsh-focus;
+  border: $bdrs-focus;
+}
```

Comment:
Why don't you pass border and shadow as parameter to do it more generic an put them as default for example:
sui-focus(shadow:$bxsh-focus, border:$bdrs-focus)???